src/components/SideBar/index.tsx

```diff
@@ -43,6 +43,7 @@ export default class SideBar extends React.Component<SiderProps, SiderState> {
             [styles.sider]: true,
             [styles[type]]: true,
             [styles.embed]: isEmbedded,
+            [styles.collapsed]: this.state.collapsed,
```

Comment:
so we can only have the padding when collapsed, and the divider can reach across the length of the panel when open